examples/ui-demo/src/components/small-cards/SolanaCard.tsx

```diff
@@ -0,0 +1,122 @@
+import { useSigner, useSignerStatus } from "@account-kit/react";
+import { LAMPORTS_PER_SOL } from "@solana/web3.js";
+import { useMutation, useQuery } from "@tanstack/react-query";
+import { useMemo } from "react";
+import { LoadingIcon } from "../icons/loading";
+import ExternalLink from "../shared/ExternalLink";
+import { Button } from "./Button";
+import { createTransfer } from "./solana/createSolanaTransfer";
+import * as solanaNetwork from "./solana/solanaNetwork";
+
+export const SolanaCard = () => {
```

Comment:
This view should probably contain some info on how to fund the address as well to actually test this out (unless we support sponsorship)